@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -843,25 +846,33 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
 
                 try {
                     // Triangulate the footprint polyline.
-                    const triangles = earcut(vertices, holes);
+                    const triangles = earcut(vertices, holes, 3);
```

Comment:
we should double check and re-evaluate this, but it may be better to triangulate before projecting.